src/en/models-config.md

```diff
@@ -146,7 +153,7 @@ agent-stream: devel
 ```
 
 
-## Provision machines faster by disabling software upgrades
+### APT updates and upgrades
```

Comment:
I liked that the heading actually had a 'benefit'. Perhaps

APT updates and ugrades - faster machine provisioning